website/content/docs/pipelines/index.mdx

```diff
@@ -194,7 +194,7 @@ When you use the pipeline run CLI, you can run a pipeline in a different workspa
 with the `-workspace` flag.
 
 ```shell
-$ workspace pipeline run builder -workspace=dev
+$ waypoint pipeline run builder -workspace=dev
```

Comment:
Thank you!! While we're here, we should fix the flag position too. I hate that our CLI can't support flags after arguments but such is life 😅 

```suggestion
$ waypoint pipeline run -workspace=dev builder
```